libraries/browser-tracker-core/src/tracker/out_queue.ts

```diff
@@ -230,14 +205,72 @@ export function OutQueueManager(
     return typeof queue[0] === 'object' && 'evt' in queue[0];
   };
 
+  // Runs `onRequestFailure` callback if defined
+  const tryOnRequestFailure = (status: number, message: string, events: EventBatch, willRetry: boolean) => {
+    onRequestFailure?.({ events, status, message, willRetry });
```

Comment:
Nit: is this function really worth having? Seems like it's just forwarding the values to the callback.

Reply:
Oh yeah, that was from when I was trying to squeeze it under 15kb, that can go now 😄